releases/release-1.23/README.md

```diff
@@ -37,51 +37,52 @@ description: |
 
 The 1.23 release cycle is proposed as follows:
 
-- **Monday, August 23rd 2021**: Week 1 - Release cycle begins
-- **TBD**: TBD - [Enhancements Freeze](../release_phases.md#enhancements-freeze)
-- **TBD**: TBD - [Code Freeze](../release_phases.md#code-freeze)
-- **TBD**: TBD - [Test Freeze](../release_phases.md#test-freeze)
-- **TBD**: TBD - Docs must be completed and reviewed
-- **Tuesday, December 7th 2021**: Week 16 - Kubernetes v1.23.0 released
+- **Monday, August 23, 2021**: Week 1 - Release cycle begins
+- **Thursday, September 9, 2021**: Week 3 - [Enhancements Freeze](../release_phases.md#enhancements-freeze)
+- **Tuesday, November 16, 2021**: Week 13 - [Code Freeze](../release_phases.md#code-freeze)
+- **Tuesday, November 23, 2021**: Week 14 - [Test Freeze](../release_phases.md#test-freeze)
+- **Tuesday, November 30, 2021**: Week 15 - Docs must be completed and reviewed
+- **Tuesday, December 7, 2021**: Week 16 - Kubernetes v1.23.0 released
 - **TBD**: Week 17 - Release Retrospective
 
 ## Timeline
 
 | **What** | **Who** | **When** | **WEEK** | **CI SIGNAL** |
 |---|---|-------|---|---|
-| Start of Release Cycle | Lead | Mon August 23rd 2021 | week 1 | [master-blocking] |
-| Start Enhancements Tracking | Enhancements Lead | Mon August 23rd 2021| Week 1 | |
-| 1.23.0-alpha.1 released | Branch Manager ([TBD](TBD)) | Wed August 25 2021 | Week 1 | |
-| Schedule finalized | Lead | Thur August 26 2021 | week 1 | |
-| Team finalized | Lead | Fri August 27 2021 | week 1 | |
-| **Begin [Enhancements Freeze]** (23:59 PDT) | Enhancements Lead | TBD | TBD | [master-blocking], [master-informing] |
+| Start of Release Cycle | Lead | Mon August 23, 2021 | week 1 | [master-blocking] |
+| Start Enhancements Tracking | Enhancements Lead | Mon August 23, 2021| week 1 | |
+| 1.23.0-alpha.1 released | Branch Manager ([TBD](TBD)) | Wed August 25, 2021 | week 1 | |
```

Comment:
Let's remove all those `TBD`s for the branch manager, the links go into nowhere and we do not have mentioned personas anywhere else in that doc:
```suggestion
| 1.23.0-alpha.1 released | Branch Manager | Wed August 25, 2021 | week 1 | |
```